### PR TITLE
fix: language picker API & corner menu

### LINKED ISSUE #129: factor out languageList in crane
`languageList` gets used a bunch in crane and also via the `LanguagePicker` component in hummingbird.

### PR DESCRIPTION
**Pull request by maartenbusstra**
resolves #129

### CODE REVIEW

File: packages/crane/src/components/menu/LanguagePicker.js [hunk 1]
```diff
@@ -86,31 +85,34 @@ export default class LanguagePicker extends Component {
...
       showAbove,
       translations,
     } = this.props;
+    debugger;
     return (
       <Wrapper>
-        <Current backgroundColor={backgroundColor}>
-          {languageList[current] ? languageList[current].nativeName : null}
-        </Current>
+        <Current backgroundColor={backgroundColor}>{current || null}</Current>
```
Comment (markjjackson) [inline packages/crane/src/components/menu/LanguagePicker.js]: Replace the backgroundColor props with theming variables? This comment applies to all colors.

File: packages/crane/src/components/menu/LanguagePicker.js [hunk 2]
```diff
@@ -86,31 +85,34 @@ export default class LanguagePicker extends Component {
...
-                key={trans.locale}
-                backgroundColorHover={backgroundColorHover}
-                onClick={e => onClickHandler(e, trans, translations, languageList)}
-              >
-                {languageList[trans.locale] ? languageList[trans.locale].nativeName : null}
-              </Translation>
-            ))}
+          {translations.map(
+            trans =>
+              console.log(trans) || (
```
Comment (markjjackson) [inline packages/crane/src/components/menu/LanguagePicker.js]: Remove this log?

File: packages/hummingbird/src/components/Navigation/ChaptersToggle.js [hunk 3]
```diff
@@ -62,11 +62,11 @@ export default class ChaptersToggle extends React.Component {
   };
 
   render() {
-    const { locale } = this.props;
+    const { title } = this.props;
```
Comment (markjjackson) [inline packages/hummingbird/src/components/Navigation/ChaptersToggle.js]: There is currently no title prop being passed in to the component:

https://github.com/wingscms/wings/blob/6f9cb706232fb5ae183ca88c7401969e6767720c/packages/hummingbird/src/components/Navigation/Navigation.js#L177

 Translations need to be added for this copy in all languages and the correct copy passed as the title.

File: packages/crane/src/components/menu/LanguagePicker.js (hunk 2 above)
Comment (busstram) [reply packages/crane/src/components/menu/LanguagePicker.js]: whoops, ty

**Comment by markjjackson** [comment]
@busstram I committed a fix for this issue with the share menu, where the icons were not showing:

Screenshot 2019-07-24 at 11 36 41

**Pull request merged**